packages/api/src/target-group/target-groups.service.ts

```diff
@@ -106,22 +106,18 @@ export class TargetGroupsService {
         return true;
     }
 
-    async findNonMainTargetGroups({
+    async findMainTargetGroups({
```

Comment:
This should be renamed to `findTargetGroups` since it is possible to find main and also non main target groups with this function